openmc/mgxs/mdgxs.py

```diff
@@ -1733,3 +1737,614 @@ def xs_tally(self):
+        xs_header = '\tCross Sections [{0}]:'.format(self.get_units(xs_type))
+
+        # If cross section data has not been computed, only print string header
+        if self.tallies is None:
+            print(string)
+            return
+
+        string += '{0: <16}\n'.format('\tEnergy Groups:')
+        template = '{0: <12}Group {1} [{2: <10} - {3: <10}MeV]\n'
+
+        # Loop over energy groups ranges
+        for group in range(1, self.num_groups + 1):
+            bounds = self.energy_groups.get_group_bounds(group)
+            string += template.format('', group, bounds[0], bounds[1])
+
+        # Loop over all subdomains
+        for subdomain in subdomains:
+
+            if self.domain_type == 'distribcell':
+                string += \
+                    '{0: <16}=\t{1}\n'.format('\tSubdomain', subdomain)
+
+            # Loop over all Nuclides
+            for nuclide in nuclides:
+
+                # Build header for nuclide type
+                if xs_type != 'sum':
+                    string += '{0: <16}=\t{1}\n'.format('\tNuclide', nuclide)
+
+                # Build header for cross section type
+                string += '{0: <16}\n'.format(xs_header)
+
+                if self.delayed_groups != None:
+
+                    for delayed_group in self.delayed_groups:
+
+                        template = '{0: <12}Delayed Group {1}:\t'
+                        string += template.format('', delayed_group)
+                        string += '\n'
+
+                        template = '{0: <12}Group {1} -> Group {2}:\t\t'
+
+                        # Loop over incoming/outgoing energy groups ranges
+                        for in_group in range(1, self.num_groups + 1):
+                            for out_group in range(1, self.num_groups + 1):
+                                string += template.format('', in_group, out_group)
+                                average = \
+                                          self.get_xs([in_group], [out_group],
+                                                      [subdomain], [nuclide],
+                                                      xs_type=xs_type,
+                                                      value='mean',
+                                                      delayed_groups=[delayed_group])
+                                rel_err = \
+                                          self.get_xs([in_group], [out_group],
+                                                      [subdomain], [nuclide],
+                                                      xs_type=xs_type,
+                                                      value='rel_err',
+                                                      delayed_groups=[delayed_group])
+                                average = average.flatten()[0]
+                                rel_err = rel_err.flatten()[0] * 100.
+                                string += '{:1.2e} +/- {:1.2e}%'.format(average,
```

Comment:
The format specifiers give a minimum width of 1, which is not very useful considering the precision is 2. Consider changing to `{:.2e}`